test/chainsaw/integration/annotated-namespace-not-enabled/chainsaw-test.yaml

```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: annotated-namespace-not-enabled
+spec:
+  steps:
+    - name: step-00
+      try:
+        - apply:
+            file: 00-install.yaml
+        - assert:
+            file: assert-1.yaml
+      catch:
+        - podLogs:
+            selector: app=test
+        - script:
```

Comment:
```suggestion
     
```

Reply:
you mean remove the script as it does the same thing as the previous entry ?